src/lib/mediaUtil.js

```diff
@@ -1,3 +1,36 @@
+// import { notEmptyString } from './formValidators';
+import { lookupReadableMetadataName } from './explorerUtil';
+
 export const VIEW_ALL_STORIES = 'VIEW_ALL_STORIES';
 export const VIEW_REGULARLY_COLLECTED = 'VIEW_REGULARLY_COLLECTED';
 export const ALL_MEDIA = -1;
+
+/* this is set up to match the expected params within the metadataCheckboxFieldArray */
+export function importTagsFromCustomSearches(tagsObj) {
+  if (tagsObj) {
+    const updatedTagsObj = { tags: {} };
+    Object.keys(tagsObj).forEach((m) => { // for each tag
+      const vals = Object.values(tagsObj[m]).map(a => a);
```

Comment:
Is the `map` needed here? Seems like it can just be `const vals = Object.values(tagsObj[m])`

Reply:
yes, thank you, it had been setting a property of a